controllers/grid/settings/user/form/UserDetailsForm.inc.php

```diff
@@ -319,6 +320,14 @@ public function execute(...$functionParams)
                 } else {
                     $this->user->setPassword(Validation::encryptCredentials($this->user->getUsername(), $this->getData('password')));
                 }
+
+				$sessionManager = SessionManager::getManager();
```

Comment:
Looks like you've got a mix of tabs and spaces here; for `main`, indenting should be 4 spaces. (The auto-formatter should be taking care of this; maybe Captain Hook is not working to set up your git hooks?)